### PR TITLE
chore: updates hello world example to include contract verification

### PR DESCRIPTION
**Pull request by dutterbutter**
**Changes made in this pull request:**

This PR updates the `deploy.ts` script mentioned in the hello world example to programmatically verify the greeter contract.

-  Adds reference to verification deps `@matterlabs/hardhat-zksync-verify` `@nomiclabs/hardhat-etherscan`
-  Updates `hardhat.config.ts` to include verifyURL and import dep
-  Updates `deploy.ts` to include hardhat verification programmatically
-  Updates expected response from `deploy.ts` script

### CODE REVIEW

File: docs/dev/building-on-zksync/hello-world.md [hunk 1]
```diff
@@ -64,6 +65,7 @@ module.exports = {
       url: "https://testnet.era.zksync.dev",
       ethNetwork: "goerli", // RPC URL of the network (e.g. `https://goerli.infura.io/v3/<API_KEY>`)
       zksync: true,
+      verifyURL: 'https://zksync2-testnet-explorer.zksync.dev/contract_verification'  // Verification endpoint for Goerli
```
Comment (kailash-manasarovar) [inline docs/dev/building-on-zksync/hello-world.md:68]: wondering about this url?

**Review by kailash-manasarovar — CHANGES_REQUESTED** [review]
For me, the language is good. If you can confirm the URL is good, I'll do a quick run through.

File: docs/dev/building-on-zksync/hello-world.md (hunk 1 above)
Comment (dutterbutter) [reply docs/dev/building-on-zksync/hello-world.md]: The comment was misleading with the mention of Goerli. Updated. The URL is correct.

**Review by kailash-manasarovar — APPROVED** [review]
LGTM

**Review by uF4No — APPROVED** [review]

**Pull request merged**